assets/css/_sass/website/_docs-page.scss

```diff
@@ -54,9 +58,8 @@
     font-size: inherit;
     color: $grey900;
   }
-  article {
-    padding: 0 28px;
-    margin: 0;
+  .content-wrapper {
+
     ul#markdown-toc {
```

Comment:
I don't remember defining this multiple times, but this class/identifier is named both here and in _family-page.scss (and actually under com-code above, too). We should only define it once as it seems to be under the same parent class (though maybe it's `.component-wrapper` > `.component-wrapper` > `ul#markdown-toc` here?). I thought I had it in a different parent class for the family page... As it stands, though, the page links are now cyan rather than gray on the family pages, which doesn't match the design spec I had. Is that intentional?

Reply:
FWIW I prefer the new styling proposed in this PR.

Reply:
Agree -- I saw that the <article> element had css rules in several places and was unsure why.  I'll clean that up and make sure that .content-wrapper is defined in one place.  The family page component-row links becoming cyan was not intentional, but I actually like it that way now that I've seen the difference.  I'm going to consult Ty on it.